corehq/apps/domain/models.py

```diff
@@ -1157,3 +1157,11 @@ class AppReleaseModeSetting(models.Model):
     def get_settings(domain):
         domain_obj, created = AppReleaseModeSetting.objects.get_or_create(domain=domain)
         return domain_obj
+
+
+class DomainOptionToggles(models.Model):
```

Comment:
This seems analogous to the various settings on `Domain` itself, eg `case_sharing`, `secure_submissions`, `commtrack_enabled`, `restrict_superusers`, etc (likewise for `UserOptionToggles` and `CouchUser`).  I can't think of a significant conceptual difference between these things.  In my ideal world, the `Domain` model would store only things about the domain itself, with perhaps a different, larger `DomainSettings` object for arbitrary config (especially as it'd be nice to have all that in SQL).

I'm concerned that this change gives us a hybrid approach, where some settings are stored on the `Domain` object, and some on `DomainOptionToggles`, without a strong rationale for which belongs where.  I think the easiest short-term approach would be to stick this on the `Domain`, following those other examples, though I'd also be in support of an effort to clean up that model and store settings elsewhere.